playbooks/roles/custom_domains/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+edxapp_data_dir: "{{ COMMON_DATA_DIR }}/edxapp"
```

Comment:
Why not include the edxapp default in the `custom_domains` role?

Reply:
there's code doing that already: https://github.com/appsembler/configuration/blob/appsembler/ficus/master/playbooks/roles/custom_domains/tasks/main.yml#L3

but it... doesn't work in Hawthorn and I haven't been able to figure out why.

Reply:
We can live with that, until it creates a problem :sweat_smile:

 Thanks @thraxil!